src/cmd/sign.js

```diff
@@ -69,7 +69,8 @@ export default function sign(
       }
 
       let [buildResult, idFromSourceDir] = await Promise.all([
-        build({sourceDir, artifactsDir: tmpDir.path()}, {manifestData}),
+        build({sourceDir, artifactsDir: tmpDir.path()}, {manifestData,
+          showReadyMessage: false}),
```

Comment:
nitpicky request (sorry): when breaking a block into a new line, the break needs to be consistent. In this case the block is `{manifestData, showReadyMessage: false}`. You could make it break consistently in a couple different ways:

Here is an example of keeping all block elements on the same line:
````javascript
build({sourceDir, artifactsDir: tmpDir.path()}, {
  manifestData, showReadyMessage: false,
}),
````
Here is an example of moving the entire block to a new line:
````javascript
build({sourceDir, artifactsDir: tmpDir.path()}, 
      {manifestData, showReadyMessage: false}),
````
This type of consistency is hard to enforce with automatic lint rules so sorry about slowing you down to fix that. It's important to fix these kinds of things because otherwise other contributors may become distracted by the inconsistency when working on an unrelated feature.